src/main/java/com/github/princesslana/eriscasper/util/SendableImage.java

```diff
@@ -0,0 +1,19 @@
+package com.github.princesslana.eriscasper.util;
+
+import java.awt.image.BufferedImage;
+import java.io.ByteArrayOutputStream;
+import java.util.Base64;
+
+public interface SendableImage {
```

Comment:
if we're just having utility methods, the style I've gone for is class with private constructor.

Reply:
gotcha, I'll change it